resources/zib/zib-MultidisciplinaryTeamMeeting.xml

```diff
@@ -0,0 +1,196 @@
+  <id value="zib-MultidisciplinaryTeamMeeting" />
+  <url value="http://nictiz.nl/fhir/StructureDefinition/zib-MultidisciplinaryTeamMeeting" />
+  <name value="ZibMultidisciplinaryTeamMeeting" />
+  <title value="zib MultidisciplinaryTeamMeeting" />
+  <status value="draft" />
+  <publisher value="Nictiz" />
+  <contact>
+    <name value="Nictiz" />
+    <telecom>
+      <system value="email" />
+      <value value="info@nictiz.nl" />
+      <use value="work" />
+    </telecom>
+  </contact>
+  <description value="A multidisciplinary team meeting is a meeting of health care providers in which the results of examinations and treatment are discussed with the aim of formulating joint advice for the treatment of the patient. A patient can also be present. When healthcare professionals from various disciplines participate, and this is structurally organized (local, regional, sub regional or through a partnership), this is often referred to as a multi-disciplinary consultation ." />
+  <purpose value="This Encounter resource represents the Dutch [zib ('Zorginformatiebouwsteen', i.e. Health and Care Information Model) MultidisciplinaryTeamMeeting-v1.0(2020EN)](https://zibs.nl/wiki/MultidisciplinaryTeamMeeting-v1.0(2020EN))." />
+  <copyright value="Copyright and related rights waived via CC0, https://creativecommons.org/publicdomain/zero/1.0/. This does not apply to information from third parties, for example a medical terminology system. The implementer alone is responsible for identifying and obtaining any necessary licenses or authorizations to utilize third party IP in connection with the specification or otherwise." />
+  <fhirVersion value="4.0.1" />
+  <mapping>
+    <identity value="zib-multidisciplinaryteammeeting-v1.0-2020EN" />
+    <uri value="https://zibs.nl/wiki/MultidisciplinaryTeamMeeting-v1.0(2020EN)" />
+    <name value="zib MultidisciplinaryTeamMeeting-v1.0(2020EN)" />
+  </mapping>
+  <kind value="resource" />
+  <abstract value="true" />
+  <type value="Encounter" />
+  <baseDefinition value="http://hl7.org/fhir/StructureDefinition/Encounter" />
+  <derivation value="constraint" />
+  <differential>
+    <element id="Encounter">
+      <path value="Encounter" />
+      <short value="MultidisciplinaryTeamMeeting" />
+      <comment value="Zib MultidisciplinaryTeamMeeting is mapped to this Encounter profile and a profile on CarePlan (&amp;amp;amp;lt;http://nictiz.nl/fhir/StructureDefinition/zib-MultidisciplinaryTeamMeeting-CarePlanMedia&amp;amp;amp;gt;). This Encounter profile acts as the focal resource of MultidisciplinaryTeamMeeting. This profile is referenced by the CarePlan resource, which holds the 'Plan' concept (NL-CM:15.2.10), through `CarePlan.encounter`." />
+      <alias value="Patientbespreking" />
+      <mapping>
+        <identity value="zib-multidisciplinaryteammeeting-v1.0-2020EN" />
+        <map value="NL-CM:15.2.1" />
+        <comment value="MultidisciplinaryTeamMeeting" />
+      </mapping>
+    </element>
```

Comment:
Can we say something about mandatory `.status` and `.class` elements?

Looking at the mandatory `.class` element let me wonder on how well this zib matches the Encounter resource.

Reply:
Following my proposal in #97 I will add something on `.status` (try your best, otherwise 'unknown'), but because `.class` has an extensible binding, I will not add anything. It is up to the IG to guide implementers if they should use data-absent-reason.

On your last comment: it is known that this zib per definition does not match Encounter, it is some sort of preadopt of R5 where the Patient does not have to be present anymore. See the comment on `.subject`. Should this comment be on the root? Since it changes the definition of the resource slightly?

Reply:
Agreed on the `.status` and `.class` and I actually think that moving the comment to the root makes sense. It concerns multiple elements and indeed changes the definition of the whole resource slightly.

Reply:
Moving the comment to the root probably still needs to be done. Maybe we should keep at least the part 'Referencing the Patient as `.subject` does not indicate its presence at the Encounter.' also on `.subject` itself.